src/module/item/armor/sheet.ts

```diff
@@ -2,9 +2,10 @@ import { AutomaticBonusProgression as ABP } from "@actor/character/automatic-bon
 import { ItemSheetOptions } from "@item/base/sheet/sheet.ts";
 import {
     CoinsPF2e,
+    type Grade,
     MATERIAL_DATA,
-    MaterialSheetData,
-    PhysicalItemSheetData,
+    type MaterialSheetData,
+    type PhysicalItemSheetData,
```

Comment:
This isn't really necessary if it's a mix: esbuild just needs to know if it can drop the imported file entirely.